assets/css/styles.css

```diff
@@ -0,0 +1,518 @@
+* {
+  font-family: "Open Sans", sans-serif;
+}
+
+body {
+  margin: 0;
+  background: url("../../images/desert_photo.jpg") no-repeat center center fixed;
+  background-size: cover;
+}
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#header-logo {
+  height: 10rem;
+  width: 10rem;
+  border: 4px solid #fff;
+  border-radius: 50%;
+}
+
+div#hero-image-container {
+  display: flex;
+  align-items: center;
+  margin: 2%;
+}
+div#hero-image-container ul {
+  padding: 0;
+  margin-left: 0.5%;
+}
+
+.name-container {
+  color: #fff;
+  font-size: 1.8em;
+  width: 50%;
+  padding-left: 10%;
+}
+/*
+ul .hero-first-name,
+ul .hero-last-name {
+  list-style-type: none;
+}
+.hero-first-name,
+.hero-last-name {
+  color: #fff;
+}
+
+ul .hero-first-name {
+  padding: 5px;
+  font-size: 1em;
+}
+ul .hero-last-name {
+  font-weight: bold;
+  padding: 4px;
+  margin-top: 10px;
+  font-size: 1.1em;
+}
+*/
+#group {
+  display: flex;
+}
+#nav-bar ul {
+  display: inline-flex;
+  justify-content: flex-end;
+}
+
+#nav-bar a {
+  padding: 0 10px;
+}
+
+#social-media {
+  display: inline-flex;
+  margin: 0 5%;
+  align-items: center;
+}
+
+img#twitter-icon {
+  width: 15%;
+  color: #fff;
+}
+img#linkedin-icon {
+  width: 20%;
+}
+/*
+#nav-bar ul{
+  display: flex;
+  justify-content: space-between;
+
+}
+
+#nav-bar ul a{
+  padding: 0 10%;
+}
+*/
+
+.nav-buttons {
+  color: #fff;
+  font-size: 1.3em;
+  font-weight: bold;
+  list-style-type: none;
+}
+nav a:hover {
+  color: blue;
+  cursor: pointer;
+}
+
+.linkedin-icon {
+  height: 30px;

```

Comment:
You should only need one size. Like height or width. But in general if I see width or height in a css file it's a sign of doing something wrong. It means you aren't letting the natural heights of things flow.